sdk/keyvault/keyvault-certificates/src/core/keyVaultClient.ts

```diff
@@ -30,5475 +30,1506 @@ class KeyVaultClient extends KeyVaultClientContext {
-    keyName: string,
-    keyVersion: string,
     options?: coreHttp.RequestOptionsBase
-  ): Promise<Models.GetKeyResponse>;
+  ): Promise<Models.DeleteCertificateContactsResponse>;
   /**
    * @param vaultBaseUrl The vault name, for example https://myvault.vault.azure.net.
-   * @param keyName The name of the key to get.
-   * @param keyVersion Adding the version parameter retrieves a specific version of a key.
    * @param callback The callback
    */
-  getKey(
+  deleteCertificateContacts(
     vaultBaseUrl: string,
-    keyName: string,
-    keyVersion: string,
-    callback: coreHttp.ServiceCallback<Models.KeyBundle>
+    callback: coreHttp.ServiceCallback<Models.Contacts>
   ): void;
   /**
    * @param vaultBaseUrl The vault name, for example https://myvault.vault.azure.net.
-   * @param keyName The name of the key to get.
-   * @param keyVersion Adding the version parameter retrieves a specific version of a key.
    * @param options The optional parameters
    * @param callback The callback
    */
-  getKey(
+  deleteCertificateContacts(
     vaultBaseUrl: string,
-    keyName: string,
-    keyVersion: string,
     options: coreHttp.RequestOptionsBase,
-    callback: coreHttp.ServiceCallback<Models.KeyBundle>
+    callback: coreHttp.ServiceCallback<Models.Contacts>
   ): void;
-  getKey(
+  deleteCertificateContacts(
     vaultBaseUrl: string,
-    keyName: string,
-    keyVersion: string,
-    options?: coreHttp.RequestOptionsBase | coreHttp.ServiceCallback<Models.KeyBundle>,
-    callback?: coreHttp.ServiceCallback<Models.KeyBundle>
-  ): Promise<Models.GetKeyResponse> {
+    options?: coreHttp.RequestOptionsBase | coreHttp.ServiceCallback<Models.Contacts>,
+    callback?: coreHttp.ServiceCallback<Models.Contacts>
+  ): Promise<Models.DeleteCertificateContactsResponse> {
     return this.sendOperationRequest(
       {
         vaultBaseUrl,
-        keyName,
-        keyVersion,
         options
       },
-      getKeyOperationSpec,
+      deleteCertificateContactsOperationSpec,
       callback
-    ) as Promise<Models.GetKeyResponse>;
+    ) as Promise<Models.DeleteCertificateContactsResponse>;
   }
 
   /**
-   * The full key identifier, attributes, and tags are provided in the response. This operation
-   * requires the keys/list permission.
-   * @summary Retrieves a list of individual key versions with the same key name.
+   * The GetCertificateIssuers operation returns the set of certificate issuer resources in the
+   * specified key vault. This operation requires the certificates/manageissuers/getissuers
+   * permission.
+   * @summary List certificate issuers for a specified key vault.
    * @param vaultBaseUrl The vault name, for example https://myvault.vault.azure.net.
-   * @param keyName The name of the key.
    * @param [options] The optional parameters
-   * @returns Promise<Models.GetKeyVersionsResponse>
+   * @returns Promise<Models.GetCertificateIssuersResponse>
    */
-  getKeyVersions(
+  getCertificateIssuers(
```

Comment:
The diff of this file is messy. I'm guessing we're now sorting the methods differently? I want to understand. Is this expected?

Reply:
It's expected because we're removing a large number of functions from this file which are no longer part of the spec used for `@azure/keyvault-certificates`  The order of functions is the same, I think Git just can't figure out the proper way to diff them since the overall file structure has a lot of overlap.

Reply:
Got it! Thank you! This PR works well!